### PR TITLE
Invite device registration after wiping credentials in DevicePage

### PR DESCRIPTION
**Pull request by davidebriani**
This PR implements a confirmation modal to be shown in DevicePage upon successful wiping of device's credentials secret, so that a feedback is displayed together with an invite to register the device again. A link is included in the feedback, so that by clicking it the user is redirected to the registration form already filled with the correct device ID.

wiped credentials feedback

### CODE REVIEW

File: src/elm/Page/Device.elm [hunk 1]
```diff
@@ -544,7 +555,7 @@ update session msg model =
                     "Warning"
 
                 body =
-                    "This will remove the current device credential secret from Astarte, forcing the device to register again and store its new credentials secret. Continue?"
+                    Html.text "This will remove the current device credential secret from Astarte, forcing the device to register again and store its new credentials secret. Continue?"
```
Comment (Pavinati) [inline src/elm/Page/Device.elm:558]: why converting the body type from string to html?

File: src/elm/Page/Device.elm [hunk 2]
```diff
@@ -561,9 +572,32 @@ update session msg model =
...
+                title =
+                    "Success"
+
+                body =
+                    Html.p
+                        []
+                        [
+                            Html.text "The device's credentials secret was wiped from Astarte. You can now register the device again and store its new credentials secret. ",
+                            Html.a
+                                [ href <| Route.toString <| Route.Realm <| Route.RegisterDevice (Just model.deviceId) ]
```
[the remark this answers is not in the document]
Comment (Pavinati) [reply src/elm/Page/Device.elm:585]: thinking about it, having a link inside a modal doesn't make much sense, you're allowing the user to "escape" the modal without dismissing or confirming it

**Review by Pavinati — CHANGES_REQUESTED** [review]

File: src/elm/Page/Device.elm (hunk 1 above)
Comment (davidebriani) [reply src/elm/Page/Device.elm:558]: Because the ConfirmModal now accepts html content instead of a simple string, in order to a more flexible modal body.

**Review by Pavinati — APPROVED** [review]

File: src/elm/Page/Device.elm [hunk 3]
```diff
@@ -553,17 +564,40 @@ update session msg model =
...
+                    "Success"
+
+                body =
+                    Html.p
+                        []
+                        [ Html.text "The device's credentials secret was wiped from Astarte. You can "
+                        , Html.a
+                            [ href <| Route.toString <| Route.Realm <| Route.RegisterDevice (Just model.deviceId) ]
+                            [ Html.text "click here" ]
+                        , Html.text " to register the device again and store its new credentials secret."
```
Comment (bettio) [inline src/elm/Page/Device.elm:586]: maybe instead of "store" we might go for "use".

**Review by bettio — COMMENTED** [review]
let's discuss this

File: src/elm/Page/Device.elm (hunk 3 above)
Comment (davidebriani) [reply src/elm/Page/Device.elm]: I went with "retrieve" instead of "store", as @rbino suggested. Would that be ok?

**Review by bettio — APPROVED** [review]

**Pull request merged**